test/performance/odp_l2fwd.c

```diff
@@ -1243,10 +1494,17 @@ static void usage(char *progname)
 	       "  -p, --packet_copy       0: Don't copy packet (default)\n"
 	       "                          1: Create and send copy of the received packet.\n"
 	       "                             Free the original packet.\n"
-	       "  -y, --pool_per_if       0: Share a single pool between all interfaces (default)\n"
+	       "  -y, --pool_per_if       Number of packet or packet vector pool per interface.\n"
```

Comment:
I'd prefer: `	       "  -y, --pool_per_if       Create a packet (and packet vector) pool per interface.\n"`

Reply:
Looks better. Will change in v4.